code/modules/industrial_lift/tram_override_objects.dm

```diff
@@ -24,17 +24,26 @@
 	opacity = FALSE
 
 /obj/machinery/door/window/left/tram
+	icon = 'icons/obj/doors/tramdoor.dmi'
+	//The tram door will only open in transit if you have engineering access. Also for repairing and such.
+	req_access = list("engineering")
+
 /obj/machinery/door/window/right/tram
+	icon = 'icons/obj/doors/tramdoor.dmi'
+	//The tram door will only open in transit if you have engineering access. Also for repairing and such.
+	req_access = list("engineering")
```

Comment:
this change was not clearly documented. please make an argument for why the tram doors should be locked while in movement

Reply:
For the brief few seconds of tram travel, seeing the real world actions right now players are bumping the door when the tram arrives, throwing the doors out of state. By fixing the delay of the doors opening, this is eliminated. It's also like an actual door would work.

Reply:
I would much rather you just make the logic for opening on arrival more robust, unless there's a better reason to prevent midtravel jumpoffs

Reply:
Can do.